pkg/ctl/cmdutils/nodegroup.go

```diff
@@ -22,6 +22,7 @@ func AddCommonCreateNodeGroupFlags(fs *pflag.FlagSet, p *api.ProviderConfig, cfg
 	fs.IntVarP(&ng.MaxSize, "nodes-max", "M", 0, "maximum nodes in ASG")
 
 	fs.IntVarP(&ng.VolumeSize, "node-volume-size", "", 0, "Node volume size (in GB)")
+	fs.StringVarP(&ng.VolumeType, "node-volume-type", "", "", "Node volume type (one of gp2, io1, st1, sc1 or standard)")
```

Comment:
Please add a default value to what it was before, I think it has effectively defaulted to `gp2`.

Reply:
In fact, there will need to be more change for default via config file, it's gonna be easier for me to ammend your branch, unless you'd like to do it yourself (happy to explain).

Reply:
I can see that the thread is not too clear about what the actual defaults are supposed to be.